<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #deede3;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="txtFecha">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Sistema de Ventas</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid #1e362d;
    border-radius: 8px;
    padding: 10px;
    background-color: var(--jade-50);
}

QGroupBox:title {
    color: #1e362d;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding-left: 5px;
    padding-right: 5px;
    border: none;
}</string>
    </property>
    <property name="title">
     <string>Acceso al Sistema</string>
    </property>
    <widget class="QPushButton" name="btnIngresar">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>110</y>
       <width>85</width>
       <height>37</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSalir">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>85</width>
       <height>37</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c0dacb, stop: 1 #1e362d);
    color: #ffffff;
    border: 1px solid #1e362d;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #689f84, stop: 1 #96bfaa);
    border: 1px solid #96bfaa;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #2a5242, stop: 1 #32624d);
    border: 1px solid #32624d;
}</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="shortcut">
      <string>Esc</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>70</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsuario">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>30</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #deede3;
    border: 1px solid #cccccc;
    padding: 6px;
    border-radius: 4px;
    font-size: 14px;
}

QLineEdit:hover {
    border: 1px solid #689f84;
}

QLineEdit:focus {
    border: 1px solid #689f84;
    outline: none;
}

QLineEdit::placeholder {
    color: #1e362d;
    font-weight: bold;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Contrase√±a:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: var(--jade-50);
}

QLabel {
    color: var(--jade-500);
    font-size: 16px;
    font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Usuario:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tbDatos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    background-color: #f2f7f4;
    border: 1px solid #cccccc;
    selection-background-color: #96bfaa;
    selection-color: #ffffff;
}

QTableView QHeaderView {
    background-color: #c0dacb;
    color: #333333;
    border: none;
}

QTableView QHeaderView::section {
    background-color: #deede3;
    color: #333333;
    padding: 6px;
    border: 1px solid #1e362d;
}

QTableView QHeaderView::section:checked {
    background-color: #96bfaa;
    color: #ffffff;
}

QTableView::item {
    padding: 6px;
}

QTableView::item:selected {
    background-color: #96bfaa;
    color: #ffffff;
}

QTableView::item:focus {
    background-color: #96bfaa;
    color: #ffffff;
    outline: none;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
